### PR TITLE
added dnsIPRange and lanIf arguments

### PR DESCRIPTION
**Pull request by emoscardini**

### CODE REVIEW

File: router_registration.py
Comment (dariuszSki) [inline router_registration.py:1]: looking at tunnel_ip and lanIf logic, I am not sure if I can set them to be different, i.e. lanIf = eth0 but tunnel_ip = 10.10.10.10/32 (tunnel_ip  interface = lo). Am I looking at this logic incorrectly?

Comment (emoscardini) [reply router_registration.py:1]: It should allow you to set any of the three options independently(-i --tunnel_ip, --lanIf, --dnsIPRange)

Comment (dariuszSki) [reply router_registration.py:1]: I think this logic is redundant, right?

```
if args.dnsIPRange:
  ...
  if not args.tunnel_ip:
                  tunnel_ip = ziti_router_auto_enroll.get_private_address()
```

Comment (dariuszSki) [reply router_registration.py:1]: with this first
```
if args.tunnel_ip:
            ...
        else:
            tunnel_ip = ziti_router_auto_enroll.get_private_address()
```

Comment (emoscardini) [reply router_registration.py:1]: yep, I agree, cleaned it up again

**Pull request merged**